TypeCobol/Compiler/Directives/IBMCompilerOptions.cs

```diff
@@ -630,11 +721,13 @@ public IBMCompilerOptions()
             optionWordToOptionName["BUFSIZE"] = BUFSIZE; optionWordToOptionName["BUF"] = BUFSIZE;
             optionWordToOptionName["CICS"] = CICS; optionWordToOptionName["NOCICS"] = CICS;
             optionWordToOptionName["CODEPAGE"] = CODEPAGE; optionWordToOptionName["CP"] = CODEPAGE;
+            optionWordToOptionName["COPYLOC"] = COPYLOC; optionWordToOptionName["CPLC"] = COPYLOC; optionWordToOptionName["NOCOPYLOC"] = COPYLOC; optionWordToOptionName["NOCPLC"] = COPYLOC;
```

Comment:
Adding new entries in `optionWordToOptionName` dictionary requires to adapt the initial capacity.

Reply:
Increased to 194